.github/workflows/ci.yml

```diff
@@ -61,7 +61,8 @@ jobs:
       - name: test (timestamp-authority)
         if: ${{ matrix.conf.os == 'ubuntu-latest' }}
         run: |
-          wget https://github.com/sigstore/timestamp-authority/releases/download/${SIGSTORE_TIMESTAMP}/timestamp-server-linux-amd64 -O /tmp/timestamp-server
+          export SIGSTORE_TIMESTAMP_VERSION=$(grep "github.com/sigstore/timestamp-authority" .github/go.mod | awk '{print $2}')
```

Comment:
FLAG: I'm unsure of the implications here - is this a dangerous pattern in a workflow ?

Reply:
looks ok to me.

export seems unnecessary though